include/dca/phys/dca_step/cluster_solver/shared_tools/accumulation/tp/tp_accumulator.hpp

```diff
@@ -122,6 +123,10 @@ class TpAccumulator<Parameters, linalg::CPU, DT> {
     return 0;
   }
 
+  linalg::util::CudaStream* get_stream() const {
+    return nullptr;
```

Comment:
And anywhere we use the result of get_stream we check for nullptr?

Reply:
No, I was relying on the fact that we never ask to sync or do other operations on a CudaStream with CPU version of the accumulator. I made it safer in 76324771